src/util.ts

```diff
@@ -77,6 +77,19 @@ export async function sendUpdateMessage(channels: TextBasedChannel[], text: stri
     }
 }
 
+export async function replyUpdateMessage(interaction: ChatInputCommandInteraction, text: string, name: string, options?: SendUpdateMessageOptions): Promise<void> {
+    await interaction.reply({
```

Comment:
Can you refactor the logic that defines this message/embed data into a function that both functions can use? Want to avoid duplicate code